src/core/Form/StatusText/StatusText.tsx

```diff
@@ -24,10 +24,17 @@ export interface StatusTextProps extends HtmlSpanProps {
 }
 
```

Comment:
### This might be over abstraction and not needed

As the same `statusTextAriaLiveMode` and `statusText` prop is repeated over and over what about if we change it to separate type?

```tsx
// src/core/Form/types.ts
export interface StatusText {
  /** Status text to be shown below the component and hint text. Use e.g. for validation error */
  statusText?: string;

  /** Aria-live mode for the status text element
   * @default assertive
   */
  statusTextAriaLiveMode?: ariaLiveModes;
}
```

Then just extending that type instead, `SearchInput` as an example:
```tsx
// src/core/Form/SearchInput/SearchInput.tsx
export interface SearchInputProps
  extends Omit<
      HtmlInputProps,
      | 'type'
      | 'disabled'
      | 'onChange'
      | 'children'
      | 'onClick'
      | 'value'
      | 'defaultValue'
    >,
    StatusText {
	//...
}
```

Reply:
I think that's a good idea. Assuming it fits all use cases, which in this case I think it does. Should not be too big of a task to change. But I think we still need to export the granular types like AriaLiveMode as well.

Reply:
I like the idea as well. Maybe a thing for a separate ticket just to keep things clean on that front?

Reply:
Fits here as well in my opinion. Would save a round of testing without affecting issues scope much. But up to author to decide.

Reply:
That's fine as well :) I'll do it while I'm at it.

Reply:
Implemented as `StatusTextCommonProps` interface.